INSTALL.md

```diff
@@ -15,18 +15,32 @@ and under [Arch/Manjaro Linux](#preparing-for-the-installation-under-arch-linux-
 
 ### Preparing for the installation under Mac OS X
 
-NB: The method explained below is recommended for beginners.
-A more flexible, but complex, installation method is given in [INSTALL\_OPAM.md](./INSTALL_OPAM.md).
+NB: The following is a recommended simple approach.
+For more customisation of your installation, see [INSTALL\_OPAM.md](./INSTALL_OPAM.md).
 
-1. Install "Homebrew", available from http://brew.sh/.
-2. Using Homebrew, install ocaml with the following command:
+1. Install OCaml and required libraries, using your preferred package manager.  E.g. using Homebrew (http://brew.sh/), issue the command:
 ```bash
-$ brew install objective-caml ocaml-num camlp5 bash ocaml-findlib
+$ brew install ocaml opam ocaml-num ocaml-findlib
 ```
-3. Install Emacs from https://emacsformacosx.com/.
-  
-Now proceed with [Installation of ProofGeneral](#installation-of-proofgeneral-all-operating-systems) and [Installing UniMath](#installing-unimath) below.
+2. If the build script later asks you to switch to a specific OCaml version, and/or install extra packages, you can do this with:
```

Comment:
What "build script" are you referring to here?

Reply:
Oh, here's the message:
```sh
--- making .coq_makefile_input
--- making sub/coq/config/coq_config.ml because of sub/coq/configure.ml
cd sub/coq && ./configure -coqide "no" -with-doc no -local -no-custom
Your version of OCaml is 4.12.0.
You need a version of OCaml between 4.05 and 4.11.
Configuration script failed!
Makefile:76: .coq_makefile_output.conf: No such file or directory
```

In the message, it's referred to as a "configure script".

Reply:
I’ve reworded this in a new commit; hope you think it’s better now.  I don’t want to say “the configuration script” because we just tell the user to invoke `make`, not the config script specifically.